pkg/executables/kubectl.go

```diff
@@ -1222,3 +1222,27 @@ func (k *Kubectl) GetDaemonSet(ctx context.Context, name, namespace, kubeconfig
 
 	return obj, nil
 }
+
+func (k *Kubectl) GetPackagesFromKubectl(ctx context.Context, packageInstanceName, kubeconfig string, output string) (string, error) {
+	params := []string{
+		"get", "packages", kubeconfigFlag, kubeconfig, namespaceFlag, constants.EksaPackagesName,
+	}
+	params = append(params, withPackageInstanceName(packageInstanceName)...)
+	params = append(params, withOutput(output)...)
+	stdOut, err := k.Execute(ctx, params...)
+	return stdOut.String(), err
+}
+
+func withOutput(output string) []string {
```

Comment:
This helpers don't seem like they are adding much apart from putting the if conditional in the function. Is there any other reason we want to do it this way? If output and packageInstanceName can be empty strings, then we should follow the following type of format: https://github.com/aws/eks-anywhere/blob/main/pkg/executables/kubectl.go#L682

Reply:
I think it makes the code easily readable.

Reply:
Ignore my comment.

Reply:
I'm down for these methods but it does seem like they are the same as `KubectlOpt`
But I do like that they are abstracted from the public interface

Reply:
Ok so I have resolved the packageInstanceName but I don't see anyway around output. I have looked into using KubectlOpt but it doesn't fit the needs for this specific case. The reason I say this is because -o flag needs to be set if and only if there is an output of type yaml or any other set. Because of that, it doesn't quite fit the KubectlOpt suggestion here. Please correct me if I am missing something.

Reply:
You only set `WithOutput` if you need it, and that would be called from the caller of `GetPackagesFromKubectl`, not from within the function.